src/dispatch/plugins/dispatch_slack/service.py

```diff
@@ -172,10 +180,11 @@ def get_user_info_by_id(client: Any, user_id: str):
     return make_call(client, "users.info", user=user_id)["user"]
 
 
-@functools.lru_cache()
+# @functools.lru_cache()
```

Comment:
I actually had to disable it, it caused me a couple of hours of confusion because it was cache the coroutine and not an actual value. I don't think we need to cache it but we can add back in a different cache if we need to.

Reply:
That was it. Pesky async